cmd/ctr/events.go

```diff
@@ -14,15 +14,23 @@ import (
 var eventsCommand = cli.Command{
 	Name:  "events",
 	Usage: "display containerd events",
+	Flags: []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "filter, f",
+			Usage: "Specify one or more event filters (i.e. type==ContainerCreate or container_id==alpine)",
```

Comment:
These filters _really need_ to follow the field path semantics.

Type is odd, as it is not really a field but `container_id` would be something like `event.container_id`, as these are related from the event field of the envelope. Filters on topic would be `topic~=/runtime/.*`, since that is a field on the envelope.

The adaptor framework does allow the flexibility of arbitrary mapping, but we should follow these conventions closely.